### DISCUSSION TITLE
Issue ID in List View in *Projects*

### DISCUSSION THREAD

**Discussion by lnt** [community/community#8491] [Projects and Issues] [answered]
When we talk to our team we share id's, to refer issue, If i have list view in front of me, i cant tell, need to click on issue to see the actual issue number.

Should be available as one column or should be able to add a column.

**Comment by dato** [comment]
Indeed. One has at least to hover to see the issue number (in the URL displayed but the the browser); but what is worse: I, at least, never get to that step, **my brain automatically assumes the row number _is_ the issue number!** (Hilarity ensues.)

It would be great to have a way of showing the issue number, either as a column or—more likely—in a non-obtrusive way in the title column itself.

Thanks for raising this issue!

P.S.: I'm talking about _Projects (beta)_ here.

**Comment by fdesmottes** [comment]
A similar issue : #9354

**Comment by SiteSeerBE** [comment]
I've been looking for a way to add the issue id to the project board. Just adding my vote to keep this question alive.

I constantly have misunderstandings with our analysts about this. 
Analyst: "Did you look at ticket 5?"
Me: "Ticket 5? That was ages ago. Oh, you means issue 579, which is now 5th on your list."

Maybe other people have other requirements. Wouldn't it be nice to be able to add any metadata as a column? Although for me personally ticket ID would already be great.

**Comment by Oakmast** [comment]
Yes not having this is a big headache for me please add thanks

**Comment by kfogel** [comment]
Same here, yes.  It's hard to have productive meetings with our team when we can't see the issue numbers in the project table view, because those numbers are the identifier we always use to quickly recognize & verbally refer to specific issues!  (One can't use the issue summary for that, after all -- it's far too long.)  The ID is a crucial piece of information.

**Comment by rileybroughten** [comment, marked as answer]
Making this as a duplicate of [#5904](https://github.com/community/community/discussions/5904), keep an eye on the original discussion post for updates! 👀

**Comment by kaiyuan01** [comment]
Agreed, not having this issue ID is a big headache for us for tracking all issues, please add thanks
